specification/cognitiveservices/data-plane/CustomSearch/readme.md

```diff
@@ -0,0 +1,32 @@
+# Cognitive Services CustomSearch SDK
+
+> see https://aka.ms/autorest
+
+Configuration for generating CustomSearch SDK.
+
+The current release is `release_1_0`.
+
+``` yaml
+
+tag: release_1_0
+openapi-type: data-plane
```

Comment:
Please add `add-credentials: true` here and not in C# section, this is a cross SDK parameter